src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    private final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        if (names == null || data == null || dateFrom == null || dateTo == null) {
+            return "";
+        }
+        StringBuilder sb = new StringBuilder();
```

Comment:
```suggestion
        StringBuilder report = new StringBuilder();
```
sb - не інформативно